CONTRIBUTING.md

```diff
@@ -32,12 +32,30 @@ source venv/bin/activate
 
 ### Testing
 
+Test environments rely on the `JUJU_VERSION` environment variable to be set to your major juju version before running. To do this, run the following command:
+
+```shell
+source ./setup.sh
+```
```

Comment:
I am personally not in favor of this approach. I would prefer we pass the juju version as a tox `posargs` with a default set to `juju 2`, i.e: `tox -e ha-integration -- juju-version=3`

Reply:
I like your option more, and it installs the correct version if we only run the following command: 

`tox -e ha-integration -- 3`

with the configuration: 

```ini
[testenv:ha-integration]
description = Run HA integration tests
pass_env =
    {[testenv]pass_env}
    CI
    CI_PACKED_CHARMS
    JUJU_VERSION
deps =
    -r {tox_root}/requirements_juju{posargs:2}.txt
    -r {tox_root}/requirements_integration.txt
    -r {tox_root}/requirements.txt
commands =
    pytest -v --tb native --log-cli-level=INFO -s {posargs} {[vars]tests_path}/integration/ha/test_ha.py
```

However, as you can see in the last line of this config, we also have `posargs` in our pytest command so this tox run tries to run tests in a file named `3`, which then fails. I've looked for ways to pass arguments to tox in a normal way, and I'm coming up short - I'll come back to it next week.